doc/README.md

```diff
@@ -1,34 +1,49 @@
 # GPflow documentation
 
-## Online
+## Read documentation online
 
 [![Documentation Status (master)](https://readthedocs.org/projects/gpflow/badge/?version=master)](http://gpflow.readthedocs.io/en/master/?badge=master)
 [![Documentation Status (develop)](https://readthedocs.org/projects/gpflow/badge/?version=develop)](http://gpflow.readthedocs.io/en/develop/?badge=develop)
 
-We use readthedocs to build the online documentation, and have separate versions for the `master` and `develop` branches: https://gpflow.readthedocs.io/en/master/ (for the latest official release e.g. on PyPI) and https://gpflow.readthedocs.io/en/develop/ (for the latest cutting-edge code available from the develop branch on github).
+We use readthedocs to build the online documentation, and have separate versions for the `master` and `develop` branches:
+https://gpflow.readthedocs.io/en/master/ (for the latest official release e.g. on PyPI) and
+https://gpflow.readthedocs.io/en/develop/ (for the latest cutting-edge code available from the develop branch on github).
 
-## Compiling documentation
+
+## Compile documentation locally
 
 To compile the GPflow documentation locally:
 
-1. Install doc dependencies
-```
-pip install sphinx sphinx_rtd_theme numpydoc nbsphinx sphinx_autodoc_typehints ipython
-```
-2. Install pandoc
-```
-pip install pandoc
-```
-If pandoc does not install via pip, or step 4 does not work, go to pandoc.org/installing.html
-
-3. Change directory to `doc`
-```
-cd doc
-```
-
-4. Compile the documentation as html
-```
-make html
-```
-
-5. Check documentation locally by opening (in a browser) doc/build/html/index.html
+1. Change to this directory (e.g. `cd doc` if you are in the GPflow git repository's base directory)
+
+2. Install doc dependencies
+   ```
+   pip install sphinx sphinx_rtd_theme numpydoc nbsphinx sphinx_autodoc_typehints ipython
```

Comment:
I was tempted to change this to
```suggestion
   pip install -r ./requirements.txt
```
but I don't like that because it has pinned versions of everything, and I don't want it to install an old version of IPython for me just so I can build docs.

Any reason we can't just have >= in the doc/requirements.txt file ?

Reply:
I guess it depends on what readthedocs does with it. Maybe worth a try, but low priority my opinion.
You could also just advice to create a different python env.

Reply:
yeah, exactly, that's why I haven't changed it yet

Reply:
cool, so let's leave it as it for now.